indigo/distribution.yaml

```diff
@@ -7270,6 +7270,11 @@ repositories:
       url: https://bitbucket.org/crl/multisense_ros
       version: default
     status: developed
+  muse_bldc_motor_drive:
+    doc:
+      type: git
+      url: https://bitbucket.org/muserobotics/muse_ros_package
```

Comment:
It was a mistake by travis account, I had been automatically flagged for potential abuse, and they will resolve this asap. I will also fix the commit tomorrow. Thanks for your help.